plugin/audit_log/tests/mtr/audit_log_rotate.test

```diff
@@ -5,10 +5,7 @@ let MYSQLD_DATADIR= $MYSQLD_DATADIR;
 
 --disable_result_log
 --disable_query_log
---source ../plugin/audit_log/tests/mtr/audit_log_events.inc
---source ../plugin/audit_log/tests/mtr/audit_log_events.inc
---source ../plugin/audit_log/tests/mtr/audit_log_events.inc
---source ../plugin/audit_log/tests/mtr/audit_log_events.inc
+--source audit_log_events.inc
```

Comment:
Wasn't this file included here four times on purpose? The test should have failed with this change

Reply:
fixed